extension/background.js

```diff
@@ -1,6 +1,20 @@
 // @ts-nocheck
 /* global chrome, MediaRecorder, FileReader */
 
+// need to reload extension when puppeteer-extra plugin is provided
+ chrome.storage.local.get(/* String or Array */["done"], function(items){
```

Comment:
Who changes the storage to done?

Reply:
to keep track that only reloading once. 

ie: only call  			`chrome.runtime.reload ()` one time. I dont know if there's better way